functions/Measure-DbaDiskSpaceRequirement.ps1

```diff
@@ -70,186 +76,224 @@ Function Measure-DbaDiskSpaceRequirement {
             Using a SQL table. We are DBA after all!
     #>
     [CmdletBinding()]
-    Param(
-        [Parameter(Mandatory,ValueFromPipelineByPropertyName)]
+    param(
+        [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
         [DbaInstanceParameter]$Source,
-        [Parameter(Mandatory,ValueFromPipelineByPropertyName)]
+        [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
         [string]$SourceDatabase,
```

Comment:
Please change to just Database - I did a mass replace for SourceDatabase -> Database and it worked well, including docs